### ISSUE TITLE
Refactor credentials section to use env variables.

### ISSUE THREAD

**Issue by wtrocki** [redhat-developer/app-services-cli#39]
TL;DR - single file produced (.kafka.env or something like it)
Contains environment variables that can be referenced:

URL 
USERNAME
PASSWORD

## Justification

We embeed credentials in properties at the moment.
that is good for local development.. but terrible for any deployment as one do not want to have credentials embedded in container etc. (classical 12 factor app thing tells that this is very bad)
I think we can change our credentials command to produce more than single file
It will always produce an environment file (that can be later easily translated into Operator/CR secret with the same variables)
Then separate quarkus properties files that reference it
